extension/controller/backgroundScript.js

```diff
@@ -293,6 +307,25 @@ const messageListener = function(message, sender) {
             translatingAsBrowse: message.translatingAsBrowse
           });
           break;
+        case "errorCollectionConsent":
+          browser.storage.local.set({ errorCollectionConsent: message.consent });
+          Sentry.getCurrentHub().getClient()
+            .getOptions().enabled = message.consent;
+          for (const [key,] of telemetryByTab) {
+            browser.tabs.sendMessage(
+              key,
+              {
+                command: "errorCollectionConsent",
+                consent: message.consent
+              }
+            );
+          }
+          break;
+        case "telemetryCollectionConsent":
+          browser.storage.local.set({ telemetryCollectionConsent: message.consent });
+          pingSender._setUploadEnabled(message.consent);
+          settings.uploadTelemetry = message.consent;
```

Comment:
do not change settings, it blocks sending deletion ping because PingSender thinks it's a debugging build

Reply:
fixed